pcsx2/CDVD/CDVD.cpp

```diff
@@ -1222,6 +1222,7 @@ static void cdvdWrite04(u8 rt) { // NCOMMAND
 				Console.WriteLn( Color_Gray, L"CdRead: Reading Sector %07d (%03d Blocks of Size %d) at Speed=%dx",
 					cdvd.SeekToSector, cdvd.nSectors,cdvd.BlockSize,cdvd.Speed);
 
+			cdvd.PwOff = 1 << Irq_DataReady;
```

Comment:
Well if it makes you feel any better, I know very little :P I suspect that command isn't really supposed to be there, but I can't prove it, nor do I know where exactly it should be.  With DVD and CD read, I suspect the data is not classed as "ready" until the data has transferred, but I couldn't guarantee that.